SwaprHQ/swapr-dapp
# [SWA-50][CHORE] - add missing imagotypes to the Routing Through marquee

## Fixes:  [SWA-50](https://linear.app/swaprdev/issue/SWA-50/add-missing-imagotypes-to-swapr-landing-page)

# Description

* Add missing imagotypes for 0x Protocol, BiSwap and Velodrome to the Routing Through marquee

![image](https://github.com/SwaprHQ/swapr-dapp/assets/21271189/3a3e9524-3921-47ee-b501-306a21cfdbcd)

# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Swapr landing page
4) The missing imagotypes should be present in the Routing Through marquee
